stable/distribution/UPGRADE_NOTES.md

```diff
@@ -0,0 +1,64 @@
+* Note the following **PostgreSQL** Helm chart changes
+  * The chart configuration has changed! See [values.yaml](values.yaml) for the new keys used
+  * **PostgreSQL** is deployed as a StatefulSet
+  * See [PostgreSQL helm chart](https://hub.helm.sh/charts/stable/postgresql) for all available configurations
+* Upgrade
+  * Due to breaking changes in the **PostgreSQL** Helm chart, a migration of the database is needed from the old to the new database
+  * The recommended migration process Full DB export and import of Postgresql
+    * Upgrade steps:
+      1. Prerequisite step to get details of existing chart\
+       a. Block user access to Distribution (do not shutdown).\
+       b. Obtain the service name (OLD_PG_SERVICE_NAME) using below command\
+          Example: OLD_PG_SERVICE_NAME value is `<OLD_RELEASE_NAME>-postgresql`
+          ```bash
+          $ kubectl get svc
+          NAME                                       TYPE           CLUSTER-IP       EXTERNAL-IP   PORT(S)                       AGE
+          <OLD_RELEASE_NAME>-postgresql              ClusterIP      10.101.250.74    <none>        5432/TCP                      114m
+          <OLD_RELEASE_NAME>-distribution            ClusterIP      10.101.250.89    <none>        80:30291/TCP                  113m
+         ```
+         c. Keep the previous password (OLD_PG_PASSWORD) or Extract them from the secret of the existing postgresql pod
+          Example: 
+          ```bash
+          OLD_PG_PASSWORD=$(kubectl get secret -n <namespace> <OLD_RELEASE_NAME>-postgresql -o jsonpath="{.data.postgres-password}" | base64 --decode)
+          ```
+         d. Stop old Distribution pod (scale down replicas to 0). Postgresql still exists
+          ```bash
+          $ kubectl scale statefulsets <REPLACE_OLD_RELEASE_NAME>-distribution --replicas=0
+          ```
+      2. Run the `helm install` with the `new version` say `distribution-new` with distribution scale down replicas to 0, Both new Postgresql and MongoDB pods starts
+          Example:
+          ```bash
+          helm install distribution-new --set replicaCount=0 jfrog/distribution
+          ```
+      3. To Migrate Postgresql data between old and new pods\
+          a. Connect to the new PostgreSQL pod (you can obtain the name by running kubectl get pods)
+           ```bash
+           $ kubectl exec -it <NAME> bash
+           ```
+          b. Once logged in, create a dump file from the previous database using pg_dump, connect to previous postgresql chart:\
+           ```bash
+           $ pg_dump -h <OLD_PG_SERVICE_NAME> -U distribution DATABASE_NAME > /tmp/backup.sql
+           ```
+          c. After you ran above command you should be prompted for a password, this password is the previous chart password (OLD_PG_PASSWORD). This operation could take some time depending on the database size.\
+          d. Once you have the backup file, you can restore it with a command like the one below:\
```

Comment:
Please add a step indicating that you need to copy the dump file to the new pod or specify the host with -H

Reply:
Already added `pg_dump -h` in the step b.

 Once logged in, create a dump file from the previous database using pg_dump, connect to previous postgresql chart:\
           $ pg_dump -h <OLD_PG_SERVICE_NAME> -U distribution DATABASE_NAME > /tmp/backup.sql